profile.py

```diff
@@ -150,8 +172,10 @@ def right_penumbra(self, upper=0.9, lower=0.1):
     def center(self, level=0.5):
         return 0.5 * (self.x_at_y(level) + self.x_at_y(level, reverse=True))
 
-    def mirror(self):
-        self.y = self.y[::-1]
+    def mirror(self, m=0):
+        # self.y = 2*m - self.y
+        self.x = 2*m - self.x
+        self.fix_profile()
```

Comment:
Of course you are right, I don't know why I got confused about this one. Sure, since "breaking" the profile is done by invering the array, fixing can be done by flipping it again. Right now I am on the way to Krakow, but when I arrive I'm going to correct the code. Also I think for now I will leave fix_profile function. It is not used anywhere right now, but might be useful in the future.